iceoryx2-bb/testing/src/assert.rs

```diff
@@ -173,6 +173,34 @@ macro_rules! assert_that {
             }
         }
     };
+    ($lhs:expr, contains_match |$element:ident| $predicate:expr) => {
```

Comment:
@elfenpiff To use `assert_that` I had to extend it with this. Thoughts?

Reply:
I like it. But a look at this file reminds me that we have to start to document this at some point ...